birdset/modules/models/wav2vec2.py

```diff
@@ -89,6 +104,27 @@ def forward(self, input_values, attention_mask=None, labels=None, return_hidden_
 
         return output
 
+    def get_embeddings(
+        self, input_tensor
+    ) -> Tuple[torch.Tensor, torch.Tensor]:
+        input_tensor = input_tensor.squeeze(1)
+
+        outputs = self.model(
+            input_tensor,
+            output_attentions=False,
+            output_hidden_states=True,
+            return_dict=True,
+            #labels=None
+        )
+        logits = outputs['logits']
+
+        last_hidden_state = outputs["hidden_states"][-1] #(batch, sequence, dim)
+        cls_state = last_hidden_state[:,0,:]
+        
+        
+        
+        return cls_state, logits
```

Comment:
only return the `cls_state`.

Reply:
Why? All of our get_embeddings() function return both because the original function had that in perch (We don't use it though)

Reply:
lets change it to only return the embeddings, to get the logits we can use the normal forward function

Reply:
I tried to do it but then noticed that the perch forward wont work anymore then...